boards/nxp/mimxrt1180_evk/doc/index.rst

```diff
@@ -109,6 +109,8 @@ configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | ACMP      | on-chip    | analog comparator                   |
 +-----------+------------+-------------------------------------+
+| CAN       | on-chip    | flexcan                             |
```

Comment:
driver/component is just `can` not flexcan

Reply:
@lucien-nxp can you address this please